cmd/object-api-datatypes.go

```diff
@@ -59,6 +59,12 @@ type BucketInfo struct {
 	Created time.Time
 }
 
+// HealInfo - represents erasure coded chunks in disks
+// TBD: Final choice of datatype
+type HealInfo struct {
+	Shards []bool
```

Comment:
This is too low level for `mc` to decide - mc cannot know what a shard means -  server itself should send some meaningful info.  For example like below

```
type StatusType int 

const (
      CanHeal                  StatusType = 0 (Object can be healed)
      Corrupted                StatusType = 1 (Cannot heal)
      QuorumUnavailabe StatusType = 2 (Can attempt heal if quorum is available)
      /// Add more states if necessary. 
)

type HealInfo struct {
      Status                       StatusType
      MissingDataCount    int
      MissingPartityCount int
      HealSize                   int64 (total amount of data + parity to be healed in size)
}
```

`Corrupted` state signifies if `xl.json` is missing from quorum disks,  `xl.json` is corrupted cannot be unmarshalled on any one of the quorum disks.
`QuorumUnavailable` state signifies if insufficient quorum present i.e to many `errDiskNotFound`

`CanHeal` means `xl.json` needs to be healed, along with healing other parts if needed. or just parts need to be healed.